hayeon/programmers-1/정수-제곱근-판별.js

```diff
@@ -0,0 +1,37 @@
+function solution(n) {
+  const int = Math.floor(Math.sqrt(n));
+  const doubleNum = Math.pow(int, 2);
+
+  if (n === doubleNum) {
+    return Math.pow(int + 1, 2);
+  } else {
+    return -1;
+  }
+}
```

Comment:
```suggestion
function solution(n) {
  const int = Math.floor(Math.sqrt(n));
  const squareNum = int ** 2

  if (n === squareNum) {
    return (int + 1) ** 2
  }
  return -1;
}
```
- 잘 푸셨습니다 👍 `Math.pow`말고 `**` 연산자를 활용해서 제곱을 구할 수 있습니다. 훨씬 코드도 깔끔해지는 것 같습니다 :) 
- 그리고 불필요한 else 블록은 제거해주세요 ..!!!! 
- Number.isInteger를 활용한 윤우님 코드 꼭 참고해보세요 ~!

Reply:
오오,, ** 연산자도 쓸 수 있었네요!!!!! 대박,, 아예 생각조차 못한 부분입니다ㅠㅠ 감사해요!! 윤우님 코드도 참고해보겠습니다!